plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -1404,6 +1432,11 @@ private static <K, V> Cache<K, AtomicReference<V>> buildBulkCache(
         return cacheBuilder.build();
     }
 
+    private enum SingletonCacheKey
```

Comment:
the placement looks arbitrary. should this be at the top or bottom?

Reply:
I placed it after all private methods but before the ones with Managed annotation

Reply:
just noticed in plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/recording/HiveMetastoreRecording.java you placed equivalent enum at the bottom

whatever